src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -363,7 +363,7 @@ protected function getQueryBuilderBetweenDatesGroupByDate(\DateTime $from, \Date
         $queryBuilder = $this->getEntityManager()->getConnection()->createQueryBuilder();
 
         return $queryBuilder
-            ->from('sylius_order', 'o')
+            ->from($this->getClassMetadata($this->_entityName)->getTableName(), 'o')
```

Comment:
Hmmm, what's the reason for using query builder from entity manager not that used elsewhere in this repository?